components/atom/card/src/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomCard'
+const CLASS_LINK = `${BASE_CLASS}-link`
+const CLASS_MEDIA = `${BASE_CLASS}-media`
+const CLASS_INFO = `${BASE_CLASS}-info`
+
+const AtomCardDiv = ({
+  className, // eslint-disable-line react/prop-types
+  media: Media, // eslint-disable-line react/prop-types
+  content: Content, // eslint-disable-line react/prop-types
+  ...props
+}) => (
+  <div className={className} {...props}>
+    <div className={CLASS_MEDIA}>
+      <Media />
+    </div>
+    <div className={CLASS_INFO}>
+      <Content />
+    </div>
+  </div>
+)
+
+const AtomCard = ({tabindex, vertical, highlight, href, ...props}) => {
+  const className = cx(
+    BASE_CLASS,
+    vertical && `${BASE_CLASS}-vertical`,
+    highlight && `${BASE_CLASS}-highlight`
+  )
+
+  if (href)
+    return (
+      <a href={href} tabIndex={tabindex} className={CLASS_LINK}>
```

Comment:
please use a linkFactory

Reply:
Refactored to not use a LinkWrapper because of React doesn't allow links inside links and the content area it may contain some links